rasa/shared/nlu/constants.py

```diff
@@ -24,3 +24,5 @@
 ENTITY_ATTRIBUTE_START = "start"
 ENTITY_ATTRIBUTE_END = "end"
 NO_ENTITY_TAG = "O"
+
+UTTER_PREFIX = "utter_"
```

Comment:
This constant had to be re-added here because `shared` module cannot import anything from outside of it(pytest fails). Need to use this constant inside `rasa.shared.nlu.training_data/util.py`.

Reply:
Let's move it to `rasa.shared` then instead of re-adding this constant 👍